src/step07.rb

```diff
@@ -9,38 +9,39 @@ module Step07
 
   def self.create_dialog
     html_file = File.join(__dir__, 'html', 'step07.html')
+    width = 15 + 128 + 15 + 15 + 128 + 15 # Margins and contents
     options = {
       :dialog_title => "Material",
-      :preferences_key => "example.htmldialog.materialinspector",
+      :preferences_key => "com.sketchup.example.htmldialog.materialinspector",
       :style => UI::HtmlDialog::STYLE_DIALOG,
       # Set a fixed size now that we know the content size.
-      :resizable => false,
-      :width => 350,
-      :height => 530,
+      :resizable => true,
+      :width => width,
+      :height => 500,
+      :use_content_size => true
```

Comment:
Should we account for older SU versions that might need some extra space when content size is not set?

Reply:
Good catch. Adding some extra space.